src/VirtualClient/VirtualClient.Common/VirtualClient.Common.csproj

```diff
@@ -1,20 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net8.0</TargetFramework>
+        <TargetFramework>net9.0</TargetFramework>
         <AssemblyName>VirtualClient.Common</AssemblyName>
         <NoWarn></NoWarn>
     </PropertyGroup>
 
     <ItemGroup>
+        <!-- Package dependency only for force updating version for severity vulnerability. -->
+        <PackageReference Include="System.Net.Http" Version="4.3.4"/>
```

Comment:
We should still place the versions in the Module.props to be consistent with our conventions for managing versions in 1 place.